docs-content/general/6_product-features/2_error-monitoring/enhancing-errors-with-github.md

```diff
@@ -0,0 +1,61 @@
+context, as well as other enhancements such as link directly to a file in GitHub and attributing errors to file changes.
+
+
+In order to turn on GitHub Enhancements, 3 actions need to be completed for your project:
+<ol>
+  <li>1. Add the GitHub Integration to Highlight</li>
+  <li>2. Create a service via the SDK</li>
+  <li>3. Link your service to a GitHub repo</li>
+</ol>
+
+## Add the GitHub Integration to Highlight
+Enable GitHub on Highlight by going to the [integrations](https://app.highlight.io/integrations) and click the "Connect" button in the GitHub section.
+
+More information on the GitHub Integration can be found at [GitHub Integration](../../7_integrations/github-integration.md).
+
+## Create a service via the SDK
+Services are created to group your logs, errors, and traces by the process that is running the code. Having a service can make it helpful to decipher
+which process caused an error, especially in code paths shared by multuple processes. They can also be used also filters for logs and traces.
+
+In order to create a new service, a service name must be added to your SDK. Reference the [SDK start up guides](../../../getting-started/1_overview.md) for more help.
+
+For example, in Golang, it would look something like:
+```
+highlight.SetProjectID("<YOUR_PROJECT_ID>")
+highlight.Start(
+    highlight.WithServiceName("my-app"),
+    highlight.WithServiceVersion("git-sha"),
+)
+defer highlight.Stop()
+```
+
+<b>Note:</b> There is also a service version that is provided in the example above. It is not necessary to enable GitHub enhancements, but is recommended that this be the current GIT SHA of the deployed code to use the most accurate files. If not provided, Highlight will fallback to your current default branch (e.g. main) GIT SHA.
+
+## Link your service to a GitHub repo
+Once a service is created, it should be visible in your project settings, under the "Services" table. The last step to enable stacktrace enhancements is to link your service to a GitHub repo, the one that should be used to enhance your errors. In addition to linking the repo, there are two fields to set configure any file path mappings from your deployment process that would be needed to correctly find a file on GitHub.
+
+1. <b>Build path prefix</b> - This path prefix was added in your deployment process, and is the path in your server that contains your files. This path is not found in GitHub,
+and should be removed when fetching the file.
+2. <b>GitHub path prefix</b> - This path prefix was removed in your deployment process, and should be prepended to your files in order to correctly find the file on GitHub.
+
+An example:
+<ol>
+    <li>1. An error received has a stacktrace that points to the file `/build/main.go`.</li>
+    <li>2. The GitHub repo was selected to be the Highlight repo.</li>
+    <li>3. Highlight's deployment process, move's all files out of the `/backend` directory and into a `/build` directory.</li>
```

Comment:
```suggestion
    <li>3. Since Highlight's deployment process moves all files out of the `/backend` directory and into the `/build` directory,  we would set "Build prefix path" to `/backend` and GitHub prefix path to `/backend`</li>
```